chains/idn-consumer/node/src/service.rs

```diff
@@ -215,6 +215,7 @@ fn start_consensus(
 		collator_service,
 		authoring_duration: Duration::from_millis(2000),
 		reinitialize: false,
+		max_pov_percentage: None,
```

Comment:
do we want to look into this later to make sure this is correct? Or are we okay with it?
```suggestion
		// TODO: make sure this is the value that we want
		max_pov_percentage: None,
```